### ISSUE TITLE
Add CI

### ISSUE THREAD

**Issue by avik-pal** [tejank10/Duckietown.jl#8]
We should have some form of CI testing the differentiability of duckietown

**Comment by tejank10** [comment]
Yes, I have been thinking about this lately: for every map, find the gradient on a 2 step BPTT and ensure that it is not `nothing`.
